pkg/cidata/cidata.go

```diff
@@ -21,7 +21,7 @@ import (
 
 	"github.com/docker/go-units"
 	"github.com/lima-vm/lima/pkg/debugutil"
-	"github.com/lima-vm/lima/pkg/identifierutil"
+	instance "github.com/lima-vm/lima/pkg/instance/hostname"
```

Comment:
This is very confusing - typically the package name is the same as the directory name. You said that we already have an instance package, but you added a second instance package in this change, no?